src/sparseml/pytorch/datasets/image_classification/README.md

```diff
@@ -0,0 +1,35 @@
+### FFCV SparseML support
+
+[FFCV] or Fast Forward Computer Vision is a drop-in data-loading system that 
+can dramatically increase data throughput in model training. [SparseML] now 
+provides experimental support for FFCV with it's [ImageNet] based datasets.
```

Comment:
also missing links for sparseml and imagenet?

Reply:
Sparseml and Imagenet anchors at the end